README.md

```diff
@@ -204,7 +204,6 @@ Admin notes
     * urn:globus:auth:scope:auth.globus.org:view_identities 
     * email
 * Get Google OAuth2 creds [here](https://console.developers.google.com/apis)
-  * Note that the Google People API must be enabled.
```

Comment:
Are there any special APIs that do need to be enabled?  Or is what we're calling always enabled by Google by default?

Reply:
I think that is is enabled by default and no special apis need to be enabled. I think the RELEASE NOTES say that whats happening now is instead of using the people API to get an email address, we are using the JWT token provided at the end of the OAUTH2 login flow, which we've had all along.